src/DETHRACE/common/loading.c

```diff
@@ -2285,6 +2285,42 @@ void LoadCar(char* pCar_name, tDriver pDriver, tCar_spec* pCar_spec, int pOwner,
     }
     fclose(f);
     fclose(g);
+
+#if DETHRACE_FIX_BUGS
+#define CHECK_BINDING_INDEX(IDX)                                                                                 \
+    do {                                                                                                         \
+        if ((IDX) >= 0) {                                                                                        \
+            if (gGroove_funk_bindings[IDX] == NULL) {                                                            \
```

Comment:
or if `IDX > COUNT_OF(gGroove_funk_bindings)` ?

Reply:
No, the issue was not out-of-bounds access, but dereferencing a null pointer inside the `gGroove_funk_bindings` array.

Reply:
```suggestion
            if (IDX >= COUNT_OF(gGroove_funk_bindings) || gGroove_funk_bindings[IDX] == NULL) {                                                            \
```